formapp/app.py

```diff
@@ -856,29 +857,34 @@ def endpoint_bgm():
                         startBGM(bgm_file, bgm_volume)
                     else:
                         startBGM(bgm_file)
-                    return render_template("message.html", forward=request.path, message = 'Playing: ' + bgm_file )
+                    return 'Playing: ' + bgm_file
                 else:
-                    return render_template("message.html", forward=request.path, message = "It has only been " + str(int(time.time() - sD.bgm['lastTime'])) + " seconds since the last song started. Please wait at least 60 seconds.")
+                    return "It has only been " + str(int(time.time() - sD.bgm['lastTime'])) + " seconds since the last song started. Please wait at least 60 seconds."
             else:
-                return render_template("message.html", forward=request.path, message = "Cannot find that filename!")
+                return "Cannot find that filename!"
         elif request.args.get('voladj'):
             if request.args.get('voladj') == 'up':
                 volBGM('up')
-                return render_template("message.html", message = 'Music volume increased by one increment.' )
+                return 'Music volume increased by one increment.'
             elif request.args.get('voladj') == 'down':
                 volBGM('down')
-                return render_template("message.html", message = 'Music volume decreased by one increment.' )
+                return 'Music volume decreased by one increment.'
```

Comment:
Why is this not returning the message template?

Reply:
Returning a string makes it easier to use the information in other pages, such as showing alerts in Advanced mode.